src/modules/FtpServer.rb

```diff
@@ -949,6 +949,12 @@ def Write
     # @return [Boolean] True on success
     def Import(settings)
       settings = deep_copy(settings)
+
+      # This setting is not a part of the general UI but is needed
+      # for the AutoYaST installation and is set in the AY configuration
+      # file. So we have to add it here too. (bnc#1047232)
+      @UI_keys << "StartDaemon"
```

Comment:
Adding a new item unconditionally should not harm when the `Import` function is called twice, but I'd rather avoid that. 

And how that behaves in the interactive configuration mode? I guess the UI might display an extra option...

IMHO doing something like

```ruby
Builtins.foreach(@UI_keys + ["StartDaemon"]) do |key|
```

few lines below would be a better approach.